helm-stack/helm-test/create-fake-flows.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+# shellcheck disable=SC2143
+if [ -z "$(java -version 2>&1 | grep version | grep 11)" ]; then
+	echo "The generator requires java 11.  Unknown version:"
+	java -version 2>&1 | grep version
+	exit 1
+fi
+
+echo "downloading populator jar"
+wget --quiet -c -O /tmp/nephron-testing-bundled.jar https://www.opennms.com/~ranger/helm-test/nephron-testing-bundled-0.3.2-SNAPSHOT.jar
+
+echo "creating fake flows in elasticsearch"
+java -jar /tmp/nephron-testing-bundled.jar --esRawFlowOutput=ELASTIC_SEARCH
+curl -XPUT -H 'Content-Type: application/json' http://localhost:9200/_template/netflow -d@netflow-template.json
```

Comment:
I would setup the index template first. I am not sure how ES behaves when index definitions come after the samples.

```suggestion
echo "setup index template for netflow index"
curl -XPUT -H 'Content-Type: application/json' http://localhost:9200/_template/netflow -d@netflow-template.json
echo "creating fake flows in elasticsearch"
java -jar /tmp/nephron-testing-bundled.jar --esRawFlowOutput=ELASTIC_SEARCH
```